allensdk/brain_observatory/behavior/session_apis/data_io/behavior_ophys_json_api.py

```diff
@@ -0,0 +1,112 @@
+import logging
+from datetime import datetime
+import pytz
+
+from allensdk.brain_observatory.behavior.session_apis.data_transforms import (
+    BehaviorOphysDataXforms)
+
+
+class BehaviorOphysJsonApi(BehaviorOphysDataXforms):
+    """
+        This class is used by both Scientifica and Mesoscope ophys experiments.
+    """
+
+    def __init__(self, data):
+        self.data = data
+        self.logger = logging.getLogger(self.__class__.__name__)
+
+    def get_ophys_experiment_id(self):
+        return self.data['ophys_experiment_id']
+
+    # TODO: This should be replaced with a dict lookup after the
+    # behavior_ophys_write_nwb LIMS strategy has been updated
+    def get_behavior_session_id(self):
+        NotImplementedError()
+
+    # TODO: This should be replaced with a dict lookup after the
+    # behavior_ophys_write_nwb LIMS strategy has been updated
+    def get_ophys_session_id(self):
+        NotImplementedError()
+
+    def get_surface_2p_pixel_size_um(self):
```

Comment:
just curious
why:
```
def get_X(self):
    return self.data['X']
```
and not
```
@property
def X(self):
    return self.data['X']
```

Reply:
API classes need to provide `get_X` methods since the classes `BehaviorSession` / `BehaviorOphysSession` which use the apis expect the api to have those methods.